CMakeLists.txt

```diff
@@ -98,7 +100,7 @@ include(CPack)
 
 # Enable Google Test
 set(CUBOS_TESTS_SOURCE
```

Comment:
Maybe move the tests cmake configuration to the tests folder (like the samples) and add it with add_subdirectory)

Reply:
Yes, I agree.
The problem is related with #34 and it will be fixed soon.